Moderation-Policy.md

```diff
@@ -43,7 +43,28 @@ Any alternative Moderation Policy used for a given repository must be included i
 
 Any Post considered to be in violation of the Node.js [Code of Conduct][] is subject to Moderation.
 
-The TSC is solely responsible for deciding what constitutes inappropriate behavior that may be subject to Moderation (see: [Escalation to the TSC](#escalation-to-the-tsc)).
+The [Code of Conduct][] calls out the following as specific examples of
+unacceptable behavior:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+In addition, the following guidelines apply:
+
+* Please be kind and courteous. There's no need to be mean or rude.
+* Respect that some individuals and cultures consider the casual use of profanity and sexualized language offensive and off-putting.
+* Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+* Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+* Insulting, demeaning, or harassing anyone, in public or in private, is not welcome behavior and will be grounds for banning from the project.
+* Use of personal pronouns in code comments and documentation is to be avoided.
```

Comment:
I think the CoC never really was a good place for this statement, and the moderation policy seems even less so… can we move this to doc/STYLE_GUIDE.md in nodejs/node or so?